shared/src/persistence/elasticsearch/casePublicSearch.ts

```diff
@@ -5,24 +5,13 @@ import { search } from './searchClient';
 /**
  * casePublicSearch
  *
- * @param {object} providers the providers object containing applicationContext, countryType, petitionerName, petitionerState, yearFiledMax, yearFiledMin
+ * @param {object} providers the providers object containing applicationContext, countryType, petitionerName, petitionerState, endDate, startDate
  * @returns {object} the case data
  */
-export const casePublicSearch = async ({
-  applicationContext,
-  countryType,
-  petitionerName,
-  petitionerState,
-  yearFiledMax,
-  yearFiledMin,
-}) => {
+export const casePublicSearch = async ({ applicationContext, searchTerms }) => {
   const { commonQuery, exactMatchesQuery } = aggregateCommonQueryParams({
     applicationContext,
-    countryType,
-    petitionerName,
-    petitionerState,
-    yearFiledMax,
-    yearFiledMin,
+    ...searchTerms,
   });
```

Comment:
It looks like [this function](https://github.com/ustaxcourt/ef-cms/pull/3686/files#diff-406d891cc37977094574710e7929c5e19e2601778605496f01e8f046b2e3a278L19) doesn't take `applicationContext`. Can we remove that and just pass along `searchTerms`?